tests/integration/path_planning.cpp

```diff
@@ -0,0 +1,338 @@
+#include <chrono>
+#include <cmath>
+#include <fstream>
+#include <iostream>
+#include <memory>
+#include <nlohmann/json.hpp>
+#include <thread>
+#include <unordered_map>
+
+#include "core/mission_state.hpp"
+#include "network/gcs.hpp"
+#include "network/gcs_macros.hpp"
+#include "network/gcs_routes.hpp"
+#include "pathing/plotting.hpp"
+#include "pathing/static.hpp"
+#include "ticks/mission_prep.hpp"
+#include "ticks/mission_upload.hpp"
+#include "ticks/path_gen.hpp"
+#include "ticks/tick.hpp"
+#include "utilities/constants.hpp"
+#include "utilities/datatypes.hpp"
+#include "utilities/http.hpp"
+
```

Comment:
Not that big of a deal, but I think it could be nice to have a comment explaining what this integration test does and what files it outputs for someone looking to test path planning in the future.